app/utils/zip-agenda-files.js

```diff
@@ -21,11 +21,12 @@ async function constructArchiveName(agenda) {
   return `VR_zitting_${formattedDate}_${agendaName}_alle_punten.zip`;
 }
 
-async function fetchArchivingJob(agenda, mandateeIds, decisions= false) {
+async function fetchArchivingJob(agenda, mandateeIds, decisions= false, extensions = ["pdf"]) {
   let url = `/agendas/${agenda.id}/agendaitems/pieces/files/archive?decisions=${decisions}`;
   if (mandateeIds.length) {
     url += '&' + (new URLSearchParams({ mandateeIds }).toString());
   }
+  url += '&' + (new URLSearchParams({ extensions }).toString());
```

Comment:
I wouldn't put a default here but in the service instead so the extensions become optional and we fallback on pdf only (the requested default)
If users want other files (like docx) they need to specify it in the frontend